.github/workflows/python-bindings.yml

```diff
@@ -0,0 +1,76 @@
+on:
+  push:
+    tags:
+      - 'accesskit_python-v*'
+
+name: Publish Python bindings
+
+jobs:
+  build-wheels:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        include:
+          - os: macos-latest
+            python-arch: x64
+            rust-target: x86_64
+          - os: macos-latest
+            python-arch: x64
+            rust-target: universal2-apple-darwin
+          - os: ubuntu-latest
+            python-arch: x64
+            rust-target: x86_64
+          - os: ubuntu-latest
+            python-arch: x64
+            rust-target: i686
+            skip-wheel-installation: true
+          - os: windows-latest
+            python-arch: x64
+            rust-target: x64
+          - os: windows-latest
+            python-arch: x86
+            rust-target: x86
+
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-python@v4
+        with:
+          python-version: 3.7
```

Comment:
[Python 3.7 was EOL'd  on `2023-06-07`](https://devguide.python.org/versions/). Also, this should probably match the python version in the `release` job which could be part of the matrix.

Reply:
It was OK when I started this PR, but it should be updated indeed.